File: src/main/java/com/weeth/domain/attendance/domain/service/AttendanceDeleteService.java
package com.weeth.domain.attendance.domain.service;

import com.weeth.domain.attendance.domain.entity.Attendance;
import com.weeth.domain.attendance.domain.entity.Session;
import com.weeth.domain.attendance.domain.repository.AttendanceRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
@RequiredArgsConstructor
public class AttendanceDeleteService {

    private final AttendanceRepository attendanceRepository;

    public void deleteAll(Session session) {
        attendanceRepository.deleteAllBySession(session);
    }
}

File: src/main/kotlin/com/weeth/domain/attendance/domain/service/AttendanceDeleteService.kt
package com.weeth.domain.attendance.domain.service

import com.weeth.domain.attendance.domain.entity.Session
import com.weeth.domain.attendance.domain.repository.AttendanceRepository
import org.springframework.stereotype.Service

@Service
class AttendanceDeleteService(
    private val attendanceRepository: AttendanceRepository,
) {
    fun deleteAll(session: Session) {
        attendanceRepository.deleteAllBySession(session)
    }
}
